easybuild/easyconfigs/m/Minimac/Minimac-2013.7.17-goolf-1.4.10.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
+
+name = 'Minimac'
+version = '2013.7.17'
+versionsuffix = '-beta'
```

Comment:
Hi folks,

I try to enter into Pablo's mind and I think he would have liked to propose instead:
```
version = '2013.7.17'
versionsuffix = '-a8011a4'
```

More generally, for nasty version-less applications we should adopt an ad-hoc format like:
```
version = '20130717'
versionsuffix = '-a8011a4' # or any other hash for that matter
```

The point is:
* Make it possible to sort by temporal release
* Confine the version/naming sufficiently to make the builds truly unique.

Your takes?